src/app/zap-templates/zcl/data-model/chip/device-energy-management-mode-cluster.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="ModeTag" type="enum16">
+    <cluster code="0x009F"/>
+    <item value="0x4000" name="No Optimization"/>
```

Comment:
The mode tags don't match the spec. What am i missing here?
Mode Tag Value | Name
--                          | --
0x4000                | Local Optimization
0x4001                | Grid Optimization